cmd/kube-proxy/app/server.go

```diff
@@ -163,6 +165,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 		"A string slice of values which specify the addresses to use for NodePorts. Values may be valid IP blocks (e.g. 1.2.3.0/24, 1.2.3.4/32). The default empty string slice ([]) means to use all local addresses. This parameter is ignored if a config file is specified by --config.")
 
 	fs.BoolVar(&o.CleanupAndExit, "cleanup", o.CleanupAndExit, "If true cleanup iptables and ipvs rules and exit.")
+	fs.BoolVar(&o.InitAndExit, "init-only", o.InitAndExit, "If true make privileged configurations and exit.")
```

Comment:
To me, "make privileged configurations" sounds like it means "create a configuration which is privileged". (This confused me the first time I saw the PR title too, which seems to me to be saying "optionally only run if the config requires privileges".)

"If true, perform any initialization steps that must be done with full root privileges, and then exit. After doing this, you can run kube-proxy again with only the CAP_NET_ADMIN and CAP_SYS_RESOURCE capabilities."